src/stories/migration.mdx

```diff
@@ -22,6 +22,11 @@ Breaking Changes
   - kyn-breadcrumb-item
   - kyn-header-panel
   - kyn-header-avatar
+- Updated Form component
+  - Radio Button Group , Number Input, Text Input , Text Area
+    - Added a dedicated slot for tooltip.
+    - `label` slot has been updated to a `label` string prop
+    - `errorIcon` has been repositioned outside of the input field and is now displayed inline with the error message
```

Comment:
No need to call this out in the migration guide since it's an internal-only change.